docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -68,16 +68,34 @@ The following video provides a visual walkthrough of the options you can set in
 
 {{< docs/play title="Table Visualizations in Grafana" url="https://play.grafana.org/d/OhR1ID6Mk/" >}}
 
+### Key Features:
+- **Multi-Mode Support**: Seamlessly visualize time series data, detailed tables, annotations, or raw JSON inputs.
+- **Advanced Formatting**: Customize your data presentation with options for date formatting and value formatting.
+- **Coloring Options**: Enhance readability and insights by applying custom coloring schemes to your data points.
+- **Diverse Cell Types**: Elevate your data display with a variety of cell types, including: Colored text and backgrounds, Gauges, JSON, Images, Sparklines, and multiple combinations.
+
 ## Annotation and alert support
 
-Annotations and alerts are not currently supported in tables.
+{{% admonition type="note" %}}
+At the moment, annotations and alerts are not supported by tables.
+{{% /admonition %}}
+
+## Table visualization functions
+
+The table visualization itself provides multiple functions.
 
-## Sort column
+### Sort column
 
 Click a column title to change the sort order from default to descending to ascending. Each time you click, the sort order changes to the next option in the cycle. You can sort multiple columns by holding the `shift` key and clicking the column name.
 
 ![Sort descending](/static/img/docs/tables/sort-descending.png 'Sort descending')
 
+### Data set selector
+
+The table panel visualization will display a dropdown list at the bottom if the data queried contains multiple data sets.
+
+![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png 'Table Panel with multiple data sets')
```

Comment:
Only alt text is required. FYI, please take a look at the [image naming guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/#media-asset-file-naming-conventions) for future reference. But don't worry about updating it for this PR.
```suggestion
![Table Panel with multiple data sets](/media/docs/grafana/panels-visualizations/TablePanelMultiSet.png)
```